docs/guide/extending_ez_platform_ui.md

```diff
@@ -165,6 +167,250 @@ $menu->reorderChildren(
 );
 ```
 
+## Tabs
+
+Many elements of the back-office interface, such as System Info or Location View, are built using tabs.
+
```

Comment:
Will we have a screenshot here, in the future?

Reply:
We're talking about how the UI is built internally. A screenshot would be pointless, because it won't show anything.

Reply:
Tabs?

Reply:
Ok, makes sense, added a screenshot